<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlUsuariosForm</class>
 <widget class="QWidget" name="ControlUsuariosForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableViewUsuario"/>
   </item>
   <item>
    <widget class="QTableView" name="tableViewAdmin"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="valorLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="agregarPushButton">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
